pallets/entry/src/lib.rs

```diff
@@ -127,7 +127,28 @@ pub mod pallet {
 	/// It maps Registry Entry Identifier to Registry Entry Details.
 	#[pallet::storage]
 	pub type RegistryEntries<T: Config> =
-		StorageMap<_, Blake2_128Concat, RegistryEntryIdOf, RegistryEntryDetailsOf<T>, OptionQuery>;
+		StorageMap<
+			_, 
+			Blake2_128Concat, 
+			RegistryEntryIdOf, 
+			RegistryEntryDetailsOf<T>, 
+			OptionQuery
+		>;
+
+	/// Storage to map for Entry hashes to corresponding Registry Identifiers.
+	/// It being a storage double-map will have the Registry Entry Hash and the Registry ID
+	/// as the key, whereas the value resulted is the Registry Entry Identifier.
+    #[pallet::storage]
+    pub type HashToIdentifier<T> = 
+        StorageDoubleMap<
```

Comment:
Why cant we do single Hash to Entry? Are we allowing same hash to be anchored by different registries?

Also, if I keep registryIdentifier empty, I get all the options right? IMO, that is good enough.

Reply:
Yes same hash can be inserted by different users to different registries. So have added registry-id. 

It is entirely possible to get the identifier for the entry hash `without` knowing the registry-id.